### DISCUSSION TITLE
`@inboxsdk/core` and MV2 support going forward

### DISCUSSION THREAD

**Discussion by wegry** [InboxSDK/InboxSDK#1013] [General]
We've been mulling whether or not to support MV2 in `@inboxsdk/core` going forward. As part of #1011, we are dropping MV2 support to prevent the SDK from causing rejections by the Chrome Web Store.

Once #970 is merged and released (2.0.0), `@inboxsdk/core` will only support MV3 extensions. If there is a need from SDK users that need MV2 support in `@inboxsdk/core`, we'd be willing to work to support that use case once more.

**Comment by Decafeinato** [comment]
Hi, you mean "only support MV3 extensions" I guess? 

Thanks for your hard work

**Reply by wegry** [reply]
That's right. I've edited the OP to reflect that.
